wandb_logger.py

```diff
@@ -19,6 +23,92 @@
 wandb.init(project='catlearn', config={})
 
 
+def compute_kge_metrics(
+    cache: RelationCache[NodeType, ArrowType],
+    triplets: List[Tuple[
+        Tuple[
+            Optional[NodeType], Optional[NodeType],
```

Comment:
There is a mismatch between your type, the docstring and your usage of the function. I guess the first triplets is always the 'full' one - given call site - so those should not be optional